meta-resin-common/recipes-containers/docker/docker/docker.conf.systemd

```diff
@@ -0,0 +1,3 @@
+[Service]
+ExecStart=
+ExecStart=/usr/bin/docker daemon --log-driver=journald -s btrfs -H fd:// -H tcp://0.0.0.0:2375/
```

Comment:
Shall we not use encrypted and authenticated access to the docker daemon? Why do we allow un-encrypetd and un-authenticated access to the docker daemon?